src/TableRow.js

```diff
@@ -73,7 +73,12 @@ class TableRow extends React.Component {
   }
 
   shouldComponentUpdate(nextProps) {
-    return !!(this.props.visible || nextProps.visible);
+    const shallowCompare = (obj1, obj2) =>
```

Comment:
no need to create function and call it here. Just do what this function does normally